app/main.js

```diff
@@ -202,24 +202,14 @@ function getTrayMenu () {
     }
   }, {
     type: 'separator'
-  }, {
-    label: 'Settings',
+  })
+
+  trayMenu.push({
+    label: 'Start a break!',
     click: function () {
-      showSettingsWindow()
+      planMicrobreak(200)
```

Comment:
What do you think should happen with timers of break that are already created? I think that before calling this, we should clear those. Check `pauseMicrobreaks` function, I check there whether maybe there is some break going on already and so on and do some stuff in such a case.